mlir-tensorrt/executor/lib/Runtime/Backend/Lua/Modules/TensorRT/TensorRTModule.cpp

```diff
@@ -360,6 +421,25 @@ static Status enqueueV3Wrapper(AllocTracker &tracker,
     return getStatusWithMsg(StatusCode::InternalError,
                             "failed to set input-consumed event");
 
+  if (hasResults) {
+    int64_t* desc = reinterpret_cast<int64_t*>(*outputDescriptors);
```

Comment:
Somewhere in this file you'll have to specifically document the ABI for the output descriptors. Something with a bit more structure to wrap the `uintptr_t` would be great, e.g. `struct OutputDescriptors` could wrap the uinptr_t and provide methods for retrieving the number of results and information about each output, setting the shape/strides/ptr, etc.

Reply:
That is a good idea. I will address this in next commit.